scripts/run.sh

```diff
@@ -22,6 +22,16 @@ pip install --extra-index-url=https://pypi.anaconda.org/rapidsai-wheels-nightly/
   "scipy" \
   "dask-cuda"
 
+
+# Clone cudf repo (need dask_cudf tests)
+CUDF_VERSION="branch-25.04"
+echo "Cloning cudf@{CUDF_VERSION}"
+
+if [ ! -d "cudf" ]; then
+    git clone https://github.com/rapidsai/cudf.git --branch $CUDF_VERSION
```

Comment:
Something I'm trying to think through: the commit we get the nightly wheel from will potentially not match the commit we test against, whenever there are commits merged between the time the nightly wheel build runs and the time this code runs.

If there are any changes to the tests that require *new* code in dask_cudf, then they'll fail here.

Ideally we'd checkout the exact commit the nightly wheel was built against, but I haven't found an easy way to do that yet.

Reply:
>If there are any changes to the tests that require new code in dask_cudf, then they'll fail here.

Aha - Great point. I don't think that's a "show stopper", but we definitely want to use the same commit if we can find a way to do that.

Reply:
Agreed, not a blocker.